AutocompleteClient/FW/Logic/Track/CIOConversionTrackData.swift

```diff
@@ -15,12 +15,14 @@ public struct CIOConversionTrackData {
 
     public let searchTerm: String
     public let itemName: String?
+    public let itemID: String?
     public let sectionName: String?
     public let revenue: Int?
 
-    public init(searchTerm: String, itemName: String? = nil, sectionName: String? = nil, revenue: Int? = nil) {
+    public init(searchTerm: String, itemName: String? = nil, itemID: String? = nil, sectionName: String? = nil, revenue: Int? = nil) {
```

Comment:
Let's remove `itemName` from conversion.  We're actively trying to deprecate it.